var/spack/repos/builtin/packages/qgis/package.py

```diff
@@ -17,12 +17,15 @@ class Qgis(CMakePackage):
 
     maintainers("adamjstewart", "Sinan81")
 
+    version("3.34.0", sha256="348a2df4c4520813a319b7f72546b3823e044cacd28646ba189b56a49c7d1b5f")
     # Prefer latest long term release
     version(
-        "3.28.11",
-        sha256="c5eb703893c7f98de051c45d677c4a34b40f986db51782a4930ddefad4e193b4",
+        "3.28.12",
+        sha256="d6d0ea39ed3433d553f8b83324dc14cfa90f8caaf766fa484791df9169800f25",
         preferred=True,
     )
+    # Prefer latest long term release
```

Comment:
I don't think this comment needs to be updated, we're preferring the above line anyway